web/Web/web/judgment/views.py

```diff
@@ -56,6 +56,7 @@ def post(self, request, *args, **kwargs):
             found_ctrl_f_terms_in_title = self.request_json.get(u"found_ctrl_f_terms_in_title", None)
             found_ctrl_f_terms_in_summary = self.request_json.get(u"found_ctrl_f_terms_in_summary", None)
             found_ctrl_f_terms_in_full_doc = self.request_json.get(u"found_ctrl_f_terms_in_full_doc", None)
+            current_docview_stack_size = self.request_json.get(u"current_docview_stack_size", None)
```

Comment:
I'm just thinking if we should use a default for this as opposed to doing `self.request_json[u"current_docview_stack_size"]`. I'm inclined to keeping it this way but we should make sure it is not `None` in the if statements.

so 
`is_from_cal and current_docview_stack_size is not None and current_docview_stack_size <= 0`

Reply:
Done 👍.